README.md

```diff
@@ -61,6 +60,13 @@ Note that if you change your `setup.py`, you'll have to repeat those last two
 steps. However, you probably won't have to rebuild the package every time you
 change your plugin.
 
+Testing
+=======
+
+Hamper uses twisted's [trial][trial] unit testing system, which is an extension of
```

Comment:
You could have written this as `[trial][]`. If the link name is empty, it uses the slug.

That being said, this should be `Twisted's [Trial][]`